gammapy/datasets/make.py

```diff
@@ -64,3 +72,158 @@ def sigma_energy_theta(energy, theta, sigma):
+    	name of the observatory; a list of choices is given in `~gammapy.obs.observatory_locations`
+    n_obs : integer
+    	number of observations for the obs table
+    debug : bool
+    	show UTC times instead of seconds after the reference
+
+    Returns
+    -------
+    obs_table : `~gammapy.obs.ObservationTable`
+    	observation table
+    """
+    n_obs_start = 1
+
+    obs_table = ObservationTable()
+
+    # build a time reference as the start of 2010
+    dateref = Time('2010-01-01 00:00:00', format='iso', scale='utc')
+    # TODO: using format "iso" for `~astropy.Time` for now; eventually change it
+    # to "fits" after the next astropy stable release (>1.0) is out.
+    dateref_mjd_fra, dateref_mjd_int = np.modf(dateref.mjd)
+
+    # header
+    header = {'OBSERVATORY_NAME': observatory_name,
+              'MJDREFI': dateref_mjd_int, 'MJDREFF': dateref_mjd_fra}
+    ObservationTable.meta = header
+
+    # obs id
+    obs_id = np.arange(n_obs_start, n_obs_start + n_obs)
+    col_obs_id = Column(name='OBS_ID', data=obs_id)
+    obs_table.add_column(col_obs_id)
+
+    # TODO: `~astropy.table.Table` doesn't handle `~astropy.time.TimeDelta` columns
+    # properly see:
+    #  https://github.com/astropy/astropy/issues/3832
+    # until this issue is solved (and included into a stable release), quantity
+    # objects are used.
+
+    # obs time: 30 min
+    time_observation = TimeDelta(30.*60.*np.ones_like(col_obs_id.data), format='sec')
+    time_observation = Quantity(time_observation.sec, 'second') # converting to quantity
+    col_time_observation = Column(name='TIME_OBSERVATION', data=time_observation)
+    obs_table.add_column(col_time_observation)
+
+    # livetime: 25 min
+    time_live = TimeDelta(25.*60.*np.ones_like(col_obs_id.data), format='sec')
+    time_live = Quantity(time_live.sec, 'second') # converting to quantity
+    col_time_live = Column(name='TIME_LIVE', data=time_live)
+    obs_table.add_column(col_time_live)
+
+    # start time
+    # random points between the start of 2010 and the end of 2014
+    # using the start of 2010 as a reference time for the header of the table
+    # observations restrict to night time
+    # considering start of astronomical day at midday: implicit in setting the
+    # start of the night, when generating random night hours
+    datestart = Time('2010-01-01 00:00:00', format='iso', scale='utc')
+    dateend = Time('2015-01-01 00:00:00', format='iso', scale='utc')
+    time_start = Time((dateend.mjd - datestart.mjd)*np.random.random(len(col_obs_id)) + datestart.mjd, format='mjd', scale='utc')
+
+    # keep only the integer part (i.e. the day, not the fraction of the day)
+    time_start_f, time_start_i = np.modf(time_start.mjd)
+    time_start = Time(time_start_i, format='mjd', scale='utc')
+
+    # random generation of night hours: 6 h (from 22 h to 4 h), leaving 1/2 h time for the last run to finish
+    night_start = TimeDelta(22.*60.*60., format='sec')
```

Comment:
The whole point of Quantity and Time is that you shouldn't have to manually do `*60.*60.`.
Please use Quantity.